src/main/java/org/mule/module/xml/internal/util/FileSchemeCorrectionOutputUriResolver.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.module.xml.internal.util;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+
+import javax.xml.transform.Result;
+
+import net.sf.saxon.lib.StandardOutputResolver;
+import net.sf.saxon.trans.XPathException;
+
+public class FileSchemeCorrectionOutputUriResolver extends StandardOutputResolver {
+
+  private static final String PATH_URI_SCHEME = "file:";
+  private static final String HAS_URI_SCHEME_REGEX = "^[a-zA-Z]+:.+$";
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused "HAS_URI_SCHEME_REGEX" private field. ([squid:S1068](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS1068))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-core-extensions-mule-xml-module&pullRequest=29&issues=AWiluMgxotIXnmsUq-mJ&open=AWiluMgxotIXnmsUq-mJ)

[ISSUE_KEY]: AWiluMgxotIXnmsUq-mJ